web/src/App.test.tsx

```diff
@@ -1,9 +1,6 @@
-import { render, screen } from '@testing-library/react'
+import { render } from '@testing-library/react'
 import App from './App'
 
-test('renders learn react link', () => {
+test('renders without crashing', () => {
```

Comment:
Haha, we have high hopes for the frontend tests, I see!

Reply:
So simple, yet so hard. Starting with the basics first 😂